src/common.fypp

```diff
@@ -156,4 +156,61 @@ ${prefix + joinstr.join([line.strip() for line in txt.split("\n")]) + suffix}$
   #:endcall
 #:enddef
 
+#!
+#! Generates an array rank suffix for subarrays along a dimension
+#!
```

Comment:
Thanks for writing these nice fypp functions. Adding a usage example or at least a reference to one of the modules where they are used would be helpful for future developers to get a better idea how to use them.

Reply:
good suggestion. I'll do it.